binary_search.py

```diff
@@ -0,0 +1,17 @@
+def binary_search(list,item):
```

Comment:
add doc string
follow naming convention, so, pls rename 'list', can be confused with List()